build.gradle

```diff
@@ -117,6 +117,9 @@ subprojects {
         all*.exclude group: "log4j", module: "log4j"
         all*.resolutionStrategy*.force 'org.jboss.logging:jboss-logging:3.2.1.Final'
         all*.resolutionStrategy*.force "com.google.guava:guava:${versions.guava}"
+        all*.resolutionStrategy*.force "com.fasterxml.jackson.core:jackson-databind:${versions.jackson}"
```

Comment:
some dependencies in tests specify older version and it could happen in intellij that serialization fails